engine/src/flutter/lib/web_ui/skwasm/render_context_skia.cpp

```diff
@@ -0,0 +1,99 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "render_context.h"
+
+#include "export.h"
+#include "flutter/display_list/skia/dl_sk_dispatcher.h"
+#include "third_party/skia/include/core/SkSurface.h"
+#include "third_party/skia/include/gpu/ganesh/GrBackendSurface.h"
+#include "third_party/skia/include/gpu/ganesh/GrDirectContext.h"
+#include "third_party/skia/include/gpu/ganesh/SkSurfaceGanesh.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLBackendSurface.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLDirectContext.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLInterface.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLMakeWebGLInterface.h"
+#include "third_party/skia/include/gpu/ganesh/gl/GrGLTypes.h"
+
+#include <GLES2/gl2.h>
+#include <GLES2/gl2ext.h>
+
+SKWASM_EXPORT bool skwasm_isWimp() {
+  return false;
+}
+
+using namespace Skwasm;
+namespace {
+class SkiaRenderContext : public RenderContext {
+ public:
+  SkiaRenderContext(int sampleCount, int stencil)
```

Comment:
variable names in the file.

Reply:
Will be addressed in followup style alignment PR.